sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -44,51 +46,65 @@ export async function executeAtomXmlOperation(
   serializer: AtomXmlSerializer,
   operationOptions: OperationOptions
 ): Promise<HttpOperationResponse> {
-  if (webResource.body) {
-    const content: object = serializer.serialize(webResource.body);
-    webResource.body = stringifyXML(content, { rootName: "entry" });
-  }
-
-  if (webResource.method == "PUT") {
-    webResource.headers.set("content-length", Buffer.byteLength(webResource.body));
-  }
+  const { span, spanOptions } = createSpan(
```

Comment:
if this is using a `core-http` pipeline to make the request with `tracingPolicy` enabled, is it worth making a span at this layer, since `tracingPolicy` will already log the request?

Or does it use a custom pipeline / omit `tracingPolicy`? If so, there are some extra pieces of metadata you should set on the http span.

Reply:
It is using the core-http pipeline.
I realized I have missed adding the tracing policy to the pipeline. Added it now.

>  is it worth making a span at this layer, since tracingPolicy will already log the request?

Are you saying that we need not create the span here since we perform `sendRequest` from here?

Reply:
I mean you're free to add it wherever you like, it's more a question of is it useful to represent in trace logs to the user or not. For other clients, we only bother to instrument public operation methods and the actual HTTP requests those operations call, so the only time you see intermediate spans is when a public operation method calls another public operation method.

It's really up to your discretion if you think it's interesting to the developer trying to diagnose their app or not.

Reply:
Not spanning `executeAtomXmlOperation`.